src/components/dataList.js

```diff
@@ -53,104 +75,150 @@
             </div>
             <div className={classes.footer}>
               {(isDev || model) && !hidePagination && (
-                <Pagination totalCount={0} resultCount={take} currentPage={1} />
+                <Pagination
+                  totalCount={0}
+                  resultCount={rowsPerPage}
+                  currentPage={1}
+                />
               )}
             </div>
           </>
         );
 
+        const handleSearch = event => {
+          setSearch(event.target.value);
+        };
+
+        const isObject = item =>
+          item && typeof item === 'object' && !Array.isArray(item);
+        const deepMerge = (target, ...sources) => {
+          if (!sources.length) return target;
+          const source = sources.shift();
+
+          if (isObject(target) && isObject(source)) {
+            // eslint-disable-next-line no-restricted-syntax
```

Comment:
You can probably solve this by using Object.entries to avoid using disabling the linter.
I didn't try this one yet. For the others to come I will include examples.

Reply:
Fixed this one with Object.keys instead of Object.entries 
`Object.entries({ "name": "Jeroen" })[0]` results in `["name", "Jeroen"]` which in turn results in a key `"Jeroen"` in the returned object. `Object.keys` only returns the actual keys of the source object.